simd/src/Kokkos_SIMD_AVX2.hpp

```diff
@@ -705,7 +705,7 @@ class basic_simd<double, simd_abi::avx2_fixed_size<4>> {
       _mm256_sqrt_pd(static_cast<__m256d>(a)));
 }
 
-#ifdef __INTEL_COMPILER
+#if defined(__INTEL_COMPILER) || defined(__INTEL_LLVM_COMPILER)
```

Comment:
This is an oversight.
We dropped support for Intel Compiler Classic in #7737
We need to fix these
```console
simd/src/Kokkos_SIMD_AVX2.hpp:708:#ifdef __INTEL_COMPILER
simd/src/Kokkos_SIMD_AVX2.hpp:973:#ifdef __INTEL_COMPILER
simd/src/Kokkos_SIMD_AVX2.hpp:1248:#ifdef __INTEL_COMPILER
simd/src/Kokkos_SIMD_AVX512.hpp:459:#ifdef __INTEL_COMPILER
simd/src/Kokkos_SIMD_AVX512.hpp:730:#ifdef __INTEL_COMPILER
simd/src/Kokkos_SIMD_AVX512.hpp:1009:#ifdef __INTEL_COMPILER
simd/unit_tests/include/SIMDTesting_Utilities.hpp:61:#ifdef __INTEL_COMPILER
simd/unit_tests/include/TestSIMD_MathOps.hpp:120:#if defined(__INTEL_COMPILER) && \
```
Please do that instead
```suggestion
#ifdef __INTEL_LLVM_COMPILER
```